server/src/entity/InventoryItem.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  BaseEntity,
+  Column,
+  CreateDateColumn,
+  Entity,
+  ManyToMany,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { MenuItem } from './MenuItem';
+
+export enum itemCategory {
+  UNKNOWN = 'Unknown',
+  VEGETABLE = 'Vegetable',
+  MEAT = 'Meat',
+  DAILY = 'Daily',
+  FRUIT = 'Fruit',
+  OTHERs = 'Others',
+}
+
+@Entity()
+export class InventoryItem extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid')
```

Comment:
Does this really need an id? Wouldn't name need to be unique anyway so it should be a good key?

Reply:
I agree. DOne.